### PR TITLE
Voice response devx5021 and Python testing, compatibility

### PR DESCRIPTION
**Pull request by ilanbiala**
* Fixes VoiceResponse.conference does not pass on the `muted` parameter
* Fixes Travis-CI to properly run tests using correct version of Python
* Fixes tests to support both Python 2 and Python 3
* Drops Python 2.6 support (**Breaking Change**)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .PHONY: clean install analysis test test-install develop docs docs-install
 
 venv:
-	virtualenv venv
+	virtualenv --python=python venv
```
Comment (efossier) [inline Makefile]: whats the benefit of doing --python=python? Isnt that the default?

File: twilio/http/validation_client.py [hunk 2]
```diff
@@ -1,4 +1,7 @@
-from urlparse import urlparse
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
```
Comment (efossier) [inline twilio/http/validation_client.py]: We might want to do this in the `compat.py` file and then import `urlparse` from that so this file doesnt have to care which version of python its being run under.

File: twilio/http/validation_client.py [hunk 3]
```diff
@@ -70,7 +73,8 @@ def request(self, method, url, params=None, data=None, headers=None, auth=None,
             timeout=timeout,
         )
 
-        return Response(int(response.status_code), response.content.decode('utf-8'))
+        # return Response(int(response.status_code), response.content.decode('utf-8'))
```
Comment (efossier) [inline twilio/http/validation_client.py]: remove comment

File: twilio/jwt/access_token/grants.py [hunk 4]
```diff
@@ -10,13 +10,11 @@ def deprecated(func):
...
     @functools.wraps(func)
     def new_func(*args, **kwargs):
-        warnings.warn_explicit(
-            "Call to deprecated function {}.".format(func.__name__),
-            category=DeprecationWarning,
-            filename=func.func_code.co_filename,
-            lineno=func.func_code.co_firstlineno + 1
-        )
+        warnings.simplefilter('always', DeprecationWarning)
+        warnings.warn("Call to deprecated function {}.".format(func.__name__), category=DeprecationWarning, stacklevel=2)
```
Comment (efossier) [inline twilio/jwt/access_token/grants.py]: why did this need to change?

File: Makefile (hunk 1 above)
Comment (ilanbiala) [inline Makefile]: Adding `--python=python` uses the standard alias Travis-CI sets up in each environment, which seemed to be the simplest way to use the intended test version. I can look around for another way if you think there is one.

File: twilio/jwt/access_token/grants.py (hunk 4 above)
Comment (ilanbiala) [inline twilio/jwt/access_token/grants.py]: `func.func_code` is no longer an attribute in Python 3, and the decorator is only used on `grants.py:108`.

Comment (efossier) [inline twilio/jwt/access_token/grants.py]: ok

**Comment by ilanbiala** [comment]
Pushing fixes and ammending commits.

File: twilio/compat.py [hunk 5]
```diff
@@ -1,3 +1,5 @@
+import six
```
Comment (jingming) [inline twilio/compat.py]: Delete this.

**Pull request merged**